Mobile Buy SDK Sample Apps/Mobile Buy SDK tvOS Sample/Mobile Buy SDK tvOS Sample/ProductsViewModel.swift

```diff
@@ -58,20 +62,26 @@ extension ProductsViewModel: UICollectionViewDataSource {
     }
     
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        return self.products.count
+        return hasProducts ? self.products.count : 1
     }
     
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: ProductCollectionViewCell.reuseIdentifier, for: indexPath) as! ProductCollectionViewCell
-        let product = self.products[indexPath.row] as BUYProduct
-        if product.imagesArray().count > 0 {
-            cell.productImage.load(product.imagesArray().first, animateChange: true, completion: nil)
+        if hasProducts {
```

Comment:
you can use an if let here.  `if let product = self.products[indexPath.row] as BUYProduct`